src/migrateCalculatedMeasures.ts

```diff
@@ -1,67 +1,40 @@
 import { ContentRecord } from "@activeviam/activeui-sdk";
-import { migrateCalculatedMeasure } from "./migrateCalculatedMeasure";
+import {
+  migrateCalculatedMeasure,
+  LegacyCalculatedMeasure,
+} from "./migrateCalculatedMeasure";
 
 /**
  * Returns the converted content record tree for the calculted measures, ready to be used in ActiveUI 5.
  */
 export function migrateCalculatedMeasures(
-  legacyCalculatedMeasures: {
-    expression: string;
-    formatStringExpression: string;
-    uniqueName: string;
-    id: string;
-  }[]
+  legacyCalculatedMeasures: LegacyCalculatedMeasure[],
+  calculatedMeasuresFolder: ContentRecord
```

Comment:
Unless I am missing something there is no need to pass `calculatedMeasuresFolder`.
You're creating an entirely new entry in the `/ui` folder with our own rules, and it shouldn't be tied structure wise to what was existing in `/pivot/entitlements/cm`.
Removing the argument will be simpler and avoid confusion.

Reply:
The only reason I passed in `calculatedMeasuresFolder` here was to get the `entry` property from it, which contains required properties such as `owners` and `readers`. Indeed each measure contains the same `owners ` and `readers` property, however, the return structure is of shape:

```
{
  entry: {
    owners: [...]
    readers: [...]
  },
  children: {
    content: {
       entry: {
        owners: [...]
        readers: [...]
       }
       // Each measure contains it's own `entry` object which also contains `owners` and `readers`
       children: {...allMeasures}
    }
  }
}
```

Reply:
> The only reason I passed in calculatedMeasuresFolder here was to get the entry property from it, which contains required properties such as owners and readers

We don't need for the calculated measures root: the way I see it `/pivot/entitlements/cm` is managed by the server (you could have calculated measures in it that do not come from AUI4, but are just conveniently defined in XML by an admin), whereas the new `/ui/calculated_measures` is managed by AUI5.
And independently from that, the permissions on those folders depends on the underlying structure: like for any of our saved content type, we have `content` and `structure` under the root `calculated_measures`.
So the root must be strictly `ROLE_CS_ROOT` because we don't want a random user to delete it, but `calculated_measures/structure` must be writable by any user (that's what happens when you create a CM from the UI).